### PR TITLE
Fixed issue with exports

### PR DESCRIPTION
**Pull request by bashleigh**
#21 Imports needed relative path and build required lib dom

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "rootDir": "./src",
     "baseUrl": "./",
     "outDir": "./dist",
-    "lib": ["es2017"]
+    "lib": ["es2017", "dom"]
```
Comment (shekohex) [inline tsconfig.json]: why do we need dom ?! :open_mouth:

Comment (bashleigh) [reply tsconfig.json]: I thought you would say this! Apparently there's an issue with superagent https://github.com/DefinitelyTyped/DefinitelyTyped/issues/12044 that requires the dom library. I didn't like it but it was the quickest fix but yea. I probably shouldn't have added it. Another method is to create a file declaring the `Blob` and `XmlHttpRequest` interfaces and I didn't like that neither

Comment (shekohex) [reply tsconfig.json]: Ok, That sounds weird but ok.

Comment (bashleigh) [reply tsconfig.json]: I don't think the dom library is a good idea tbh

Comment (shekohex) [reply tsconfig.json]: me too, but anyway we need to investigate more to see what cusses this issue.
is that a typescript issue ?

**Comment by jkernech** [comment]
Any news on this PR?

**Comment by logicflakes** [comment]
Still not working, any update?

**Comment by jkernech** [comment]
Hey @bashleigh, can I do something to help out on this one?

**Comment by bashleigh** [comment]
@jkernech yes please! I'm struggling with my work load and this one is at the back of the queue. There's an issue with building the package for release. Need to remove lib that I added and figure out why the exception is thrown from tsc. Thanks! I'll do my best to come back to this!!

**Pull request closed without merging**